multimodal/image_question_answering/.env.example

```
@@ -1 +1,3 @@
-﻿GOOGLE_API_KEY=your_api_key_here
+﻿# Google AI Studio API key (required)
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The ﻿# Google AI Studio API key (required) key has incorrect delimiter

(IncorrectDelimiter)